public/docs/dart/latest/tutorial/toh-pt5.jade

```diff
@@ -48,648 +48,658 @@ p Run the #[+liveExampleLink2('', 'toh-5')] for this part.
 
-    Add a `app_component.css` file to the `app` folder with the following content.
-+makeExample('toh-5/dart/lib/app_component.css', 'css', 'lib/app_component.css (Navigation Styles)')
+  Add a `app_component.css` file to the `app` folder with the following content.
++makeExample('toh-5/dart/lib/app_component.css', '', 'lib/app_component.css (navigation styles)')
 .l-sub-section
-    :marked
-        **The *router-link-active* class**
+  :marked
+    **The *router-link-active* class**
 
-        The Angular Router adds the `router-link-active` class to the HTML navigation element
-        whose route matches the active route. All we have to do is define the style for it. Sweet!
+    The Angular Router adds the `router-link-active` class to the HTML navigation element
+    whose route matches the active route. All we have to do is define the style for it. Sweet!
 :marked
-    Set the `AppComponent`’s `styleUrls` property to this CSS file.
+  Set the `AppComponent`’s `styleUrls` property to this CSS file.
 +makeExample('toh-5/dart/lib/app_component.dart','style-urls', 'lib/app_component.dart (styleUrls)')(format=".")
 :marked
-    ### Global application styles
-    When we add styles to a component, we're keeping everything a component needs
-    &mdash; HTML, the CSS, the code &mdash; together in one convenient place.
-    It's pretty easy to package it all up and re-use the component somewhere else.
+  ### Global application styles
+  When we add styles to a component, we're keeping everything a component needs
+  &mdash; HTML, the CSS, the code &mdash; together in one convenient place.
+  It's pretty easy to package it all up and re-use the component somewhere else.
+
+  We can also create styles at the *application level* outside of any component.
 
-    We can also create styles at the *application level* outside of any component.
+  Our designers provided some basic styles to apply to elements across the entire app.
+  These correspond to the full set of master styles that we 
+  introduced earlier (see 
+  [QuickStart, "Add some style"](../quickstart.html#!#add-some-style)).
+  Here is an excerpt.
 
-    Our designers provided some basic styles to apply to elements across the entire app.
-    Add the following to a new file named `styles.css` in the root folder.
-+makeExample('toh-5/ts/styles.1.css', '', 'styles.css (App Styles)')(format=".")
++makeExample('toh-5/ts/styles.1.css', 'toh-excerpt', 'styles.css (app styles excerpt)')(format=".")
+
+- var styles_css = 'https://raw.githubusercontent.com/angular/angular.io/master/public/docs/_examples/styles.css'
 
 :marked
-    Reference this stylesheet within the `index.html` in the traditional manner.
+  Add a new file named `styles.css` in the root folder, if there isn't one already.
+  Ensure that it contains the [master styles given here](!{styles_css}).
+
+  Also ensure this stylesheet is referenced in the traditional manner within `index.html`.
```

Comment:
"is referenced" is passive voice. Maybe:

Also, edit `index.html` to refer to this stylesheet: